### DISCUSSION TITLE
Mirth connect in Docker

### DISCUSSION THREAD

**Discussion by sangrecita** [nextgenhealthcare/connect#6084] [Q&A] [answered]
Hello everyone! I am looking for information regarding a more stable version of mirth connect to build in Docker. I read suggestions or recommendations to review information available on the web. Thank you!

**Comment by jonbartels** [comment]
I do not understand your question. https://github.com/nextgenhealthcare/connect-docker is a good baseline for Docker deployments.

What have you observed or read about stability problems with Mirth? The application can fail but if there is a specific problem the community might be able to help address that specific problem.

What information did you find when you searched for your own solution?

**Reply by sangrecita** [reply]
Actually I meant that based on your experiences you could give me suggestions about which is the most stable version of mirth. The latest version according to logic should be, can you confirm it?

**Reply by pacmano1** [reply, marked as answer]
That latest version was released a few days ago and it had many library updates.   Historically releeses of Mirth are stable, but you should probably review https://github.com/nextgenhealthcare/connect/wiki for recent releases and make your own determination.    I suspect in about 30 days there will be plenty of feedback on 4.5 should it have problems.

**Reply by sangrecita** [reply]
Thank you very much @pacmano I will visit the link
